tests/Common/JobPollDelayTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Keboola\Test\Common;
+
+use Keboola\Csv\CsvFile;
+use Keboola\Test\StorageApiTestCase;
+
+class JobPollDelayTest extends StorageApiTestCase
+{
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->_initEmptyTestBuckets();
+    }
+
+    public function testSetJobPollDelay()
+    {
+        $methodUsed = false;
+
+        $linearDelay = function ($tries) use (&$methodUsed) {
```

Comment:
I think we can use generator syntax for the delay function. Not much of a difference though...

Reply:
what is generator syntax? do you have example?

Reply:
http://php.net/manual/en/language.generators.syntax.php

```php
$linearDelay = function($tries) use (&$methodUsed) {
    return (int) $tries;
};
```
becomes
```php
$linearDelay = function() use (&$methodUsed) {
    $i = 0;
    while(true) {
        $i++;
        yield $i;
    }
};
```
but it does not work, I cannot call `$linearDelay()`, it can be only used in `foreach`. Sorry for confusion, my bad.